api.swagger.yaml

```diff
@@ -16736,6 +16736,21 @@ components:
         - parent_uuid
         - body
         - author_uuid
+    comment-author-schema:
```

Comment:
Happy with the format, the only issue is that this is displaying on the POST /comments endpoint which shouldn't be allowed.